web/massastation/src/pages/Store/StoreSection/StorePlugin.tsx

```diff
@@ -36,15 +42,40 @@ function StorePlugin(props: StorePluginProps) {
     }
   }, [isInstallSuccess]);
 
+  const argsStoreError = [
+    <div
+      onMouseEnter={() => setShowTooltip(true)}
+      onMouseLeave={() => setShowTooltip(false)}
+    >
+      <FiAlertTriangle color="#FFA41D" />
+      <ToolTip showTooltip={showTooltip} content={warningMessage} />
+    </div>,
+    <FiDownload className="w-6 h-10 text-tertiary" />,
+  ];
+
   const argsStore = {
     preIcon: <img src={logo} alt="plugin-logo" />,
     topAction: <FiDownload onClick={() => mutate({ params })} />,
     title: name,
     subtitle: author,
     subtitleIcon: massalabsNomination.includes(author) ? <Certificate /> : null,
     content: description,
+    topActions: !iscompatible ? argsStoreError : undefined,
   };
 
+  function ToolTip({ ...props }) {
+    const { showTooltip, content } = props;
+    return (
+      <>
+        {showTooltip && (
+          <div className="flex flex-col w-96 absolute z-10 t-10 l-10 bg-tertiary p-3 rounded-lg text-neutral">
```

Comment:
```suggestion
          <div className="w-96 absolute z-10 t-10 l-10 bg-tertiary p-3 rounded-lg text-neutral">
```

if you just want to put the items one below the other, you don't need this. 😉